src/Attachment.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef GAZEBO_USV_ATTACHMENT_HPP
+#define GAZEBO_USV_ATTACHMENT_HPP
+
+#include <gazebo/common/UpdateInfo.hh>
+#include <gazebo/physics/PhysicsTypes.hh>
+#include <gazebo/transport/TransportTypes.hh>
+
+
+#include <ignition/math/Vector3.hh>
+
+#include "msgs.pb.h"
+
+namespace gazebo_usv {
+namespace msgs {
+    typedef const boost::shared_ptr<const DirectionedForce> DirectionedForcePtr;
+}
+    class Actuators;
+    /**
+     * @brief Gazebo World Plugin for tractive force on tethers
```

Comment:
Not only for that.

In any case, `Attachment` really does not ring right at all. DirectForceApplication ?

Reply:
fair point, DirectForceApplication looks good to me